apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageE2E.java

```diff
@@ -0,0 +1,101 @@
+package gov.cms.bfd.server.war.r4.providers;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+
+import gov.cms.bfd.server.war.ServerRequiredTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/** Endpoint end-to-end test for the V2 Coverage endpoint. */
+public class CoverageE2E extends ServerRequiredTest {
```

Comment:
The tests in this class are shared for all the others; once you've reviewed one of the test classes, the others are copy/pastes with the names, endpoint, and number of expected results changed.